<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDevise_Dialog</class>
 <widget class="QDialog" name="AddDevise_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="addDevise_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Code</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="codeEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Monnaie</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="monnaieEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="zoneEdit">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Zone g√©ographique</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addDevise_button</sender>
   <signal>accepted()</signal>
   <receiver>AddDevise_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addDevise_button</sender>
   <signal>rejected()</signal>
   <receiver>AddDevise_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
